docs/upgrade/11_to_12/workflows_mariadb.py

```diff
@@ -233,12 +235,20 @@ def parse_bool(value):
         workflow_operations = []
         workflow_operation_config = []
         workflow_config = []
+        description_limited = get_node_value(root, 'description', WORKFLOW_NS)
+        if description_limited is None:
+          description_limited = None
+        else:
+          description_limited = description_limited.lstrip().rstrip()
```

Comment:
Could use `strip()` to save on characters :D

Reply:
PR fixes a bug, and this is what you are stuck at?

Reply:
Absolutely not. I am sorry if this came across in a negatively. Probably should have made this comment seperately from the others to make it more clear that this is not something that needs to be changed to merge it, but more of a fun little optimization that is ultimately very inconsequential.